core/java/com/google/instrumentation/stats/Tag.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2016, Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.instrumentation.stats;
+
+/**
+ * A pair of consisting of an associated {@link TagKey} and {@link TagValue}.
+ */
+public final class Tag {
+  /**
+   * Constructs a new {@link Tag} from the give key and value.
+   */
+  public static Tag create(TagKey key, TagValue value) {
```

Comment:
not necessarily important here, but I've found it cluttery to have to pass strings through things like this directly. a static factory for string, string is probably more friendly (unless this isn't intended to be used by end users)